flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/FlinkCatalog.java

```diff
@@ -130,6 +131,9 @@ public void open() throws CatalogException {
       createDatabase(getDefaultDatabase(), ImmutableMap.of(), true);
     } catch (DatabaseAlreadyExistException e) {
       // Ignore the exception if it's already exist.
+    } catch (ForbiddenException e) {
```

Comment:
instead of ignoring the exception, can we just do a check on if the database exist or not?

```
public boolean databaseExists(String databaseName) throws CatalogException {
```

Reply:
That also works for me. This will introduce another call, but I think that's fine here. Updated the PR